libheif/libheif_build.py

```diff
@@ -10,7 +10,7 @@
 include_dirs = ["/usr/local/include", "/usr/include", "/opt/local/include"]
 library_dirs = ["/usr/local/lib", "/usr/lib", "/lib", "/opt/local/lib"]
 
-homebrew_prefix = os.getenv("HOMEBREW_PREFIX")
+homebrew_prefix = os.getenv("PATH")
```

Comment:
First of all, PATH environment variable specifies the directories to be searched to find a command, not a directories which contains "include" or "lib" directories. Second, PATH environment variable is a list of directories, you can't join a list with a name.